src/main/java/com/fundation/webservice/model/Metadata.java

```diff
@@ -26,7 +26,7 @@
     private final String EXIFTOOL_DIR = Directories.EXIFTOOL_DIR.getDir();
 
     /**
-     * This method writes a xmp file containing the param file metadata info returned by exiftool in XMP format.
+     * Writes a xmp file containing the param file metadata info returned by exiftool in XMP format.
```

Comment:
This might not be in accordance to the team's adopted redaction style.

AFAIR, it should somthing like: Method that writes...

Reply:
resolved

Reply:
It is now compliant.